cloud/memory-management.md

```diff
@@ -0,0 +1,27 @@
+# Memory management
+If you run intensive queries on your Timescale Cloud services, you might
+encounter out of memory (OOM) errors. This occurs if your query consumes more
+memory than is available.
+
+When this happens, an `OOM killer` process shuts down PostgreSQL processes using
+`SIGKILL` commands, until the memory usage falls below the upper limit. To
+prevent service disruption caused by OOM errors, Timescale Cloud attempts to
+shut down only the query that caused the problem. This means that the
+problematic query does not run, but that your PostgreSQL service continues to
+operate normally.
+
+OOM errors can kill the entire server process, and usually require a restart.
+When the OOM killer is triggered, you need to wait for the entire service to
+come back online before you can reconnect. The error in the log usually looks
+like this:
```

Comment:
I got confused jumping back and forth between Timescale's way and the normal OOM way. One possible way to clarify:

```suggestion
When this happens, an `OOM killer` process shuts down PostgreSQL processes using
`SIGKILL` commands, until the memory usage falls below the upper limit. Because
this kills the entire server process, it usually requires a restart. To
prevent service disruption caused by OOM errors, Timescale Cloud attempts to
shut down only the query that caused the problem. This means that the
problematic query does not run, but that your PostgreSQL service continues to
operate normally.

If the normal OOM killer is triggered, the error log looks like this:
```